client/my-sites/google-my-business/select-location/index.js

```diff
@@ -79,6 +84,8 @@ class GoogleMyBusinessSelectLocation extends Component {
 					<GoogleMyBusinessLocation key={ location.ID } location={ location } isCompact>
 						<GoogleMyBusinessSelectLocationButton
 							location={ location }
+							locationCount={ locations.length }
+							verifiedLocationCount={ verifiedLocations.length }
```

Comment:
I always get 0 here even though I have 2 locations and both are verified.
Also I don't think we should pass those variables to the `GoogleMyBusinessSelectLocationButton` just for tracking purposes. The way we do it usually is to add an optional onClick props

Reply:
Nevermind for the 0, `getAvailableExternalAccounts` was caching my GMB connections, I had to empty the store.